UserInterface/GameManager/GameManager.cs

```diff
@@ -67,45 +92,74 @@ private void RunGame()
             while (isGameOnGoing)
             {
                 Console.CursorVisible = false;
-                Pair? currentPair = _pairManager.Pair;
 
-                _board.FillBoardWithAnimals(_animals);
+                _board.FillBoardWithItems(_gameItems);
 
                 _userInterface.DrawBoard(_board);
-                _board.RemoveAnimalFromBoard(_animals);
+                _board.PrepareBoard(_gameItems);
+                
+                _movementManager.Act(_gameItems, _board);
+
+                _pairManager.RemoveNotExistingPairs(_pairs);
+               
+                _pairManager.CheckPairForExistence(_pairs, _board, _gameItems, _childrenGameItems);
 
-                _animalActions.Move(_board, _animals);
+                _pairManager.AddPairToList(_pairManager.CreatePair(_gameItems), _pairs);
+                
+                _additionManager.ProcessChildrenItems(_gameItems, _childrenGameItems);
 
-                _pairManager.RemoveNonExistsPairs(_pairs);
+                _deletionManager.RemoveInactiveItems(_gameItems);
+                Thread.Sleep(ConstantsRepository.ThreadSleep);
+                ConsoleKey? consoleKey = _userInterface.GetInputKey();
+            
+                _additionManager.ProcessAddNewItem(consoleKey, _gameItemsInfo, _gameItems, _board);    
 
-                _pairManager.CheckPairForExistence(_pairs, _board, _animals, _children);
+                isGameOnGoing = (consoleKey != ConsoleKey.Q && consoleKey != ConsoleKey.Escape);
+            }
+        }
 
-                _pairManager.CreatePair(_animals);
+        /// <summary>
+        /// Loads instances of the DLLs' children classes. 
+        /// </summary>
+        private void LoadInstancesOfDllChildrenClasses()
```

Comment:
Dictionary can consist of consolekey and Type (System class).
and there should not be need in GameItemsInfo class.

Reply:
I need other fields to represent info for user.

Reply:
Maybe then Dictionary<ConsoleKey, IItem>.
This way you can get all the information you need.
I highly believe you don't need GameItemsInfo class and Item class should be enough :D
Just looks weird to me having two classes with similar names and pretty much same parameters.

Reply:
Uhm now this method gets list and creates same list :D
In your controller you can do 
_gameItemsInfo = LoadBoardItems();
and this method is redundant now.